configs/projects/pe-installer-runtime-main.rb

```diff
@@ -0,0 +1,177 @@
+project 'pe-installer-runtime' do |proj|
```

Comment:
I'm not sure if this actually impacts anything, but all the other runtime projects have a 1:1 mapping between the filename and the project name... I think this might affect the shipping directory.

```
configs/projects/agent-runtime-1.10.x.rb:project 'agent-runtime-1.10.x' do |proj|
configs/projects/agent-runtime-5.5.x.rb:project 'agent-runtime-5.5.x' do |proj|
configs/projects/agent-runtime-6.x.rb:project 'agent-runtime-6.x' do |proj|
configs/projects/agent-runtime-main.rb:project 'agent-runtime-main' do |proj|
configs/projects/bolt-runtime.rb:project 'bolt-runtime' do |proj|
configs/projects/client-tools-runtime-2019.8.x.rb:project 'client-tools-runtime-2019.8.x' do |proj|
configs/projects/client-tools-runtime-irving.rb:project 'client-tools-runtime-irving' do |proj|
configs/projects/client-tools-runtime-main.rb:project 'client-tools-runtime-main' do |proj|
configs/projects/pdk-runtime.rb:project 'pdk-runtime' do |proj|
configs/projects/pe-bolt-server-runtime-2019.8.x.rb:project 'pe-bolt-server-runtime-2019.8.x' do |proj|
configs/projects/pe-bolt-server-runtime-main.rb:project 'pe-bolt-server-runtime-main' do |proj|
configs/projects/pe-installer-runtime.rb:project 'pe-installer-runtime' do |proj|
```

Reply:
Looking at a few of the other projects I think you're 100% right. Thanks for catching this!